examples/example.krun

```diff
@@ -73,3 +73,6 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+# Autoload rmsr kernel module (on by defualt)
+# AUTOLOAD_RMSR = True
```

Comment:
Hrm, I was imagining a config var to specify the location of rmsr. They way you locate rmsr is fine (assume the krun build system built it) is absolutel fine, but now I wonder if there would ever be a situation where `AUTOLOAD_RMSR = False` would be useful. Is there? Or should we kill the config variable altogether?